### PR TITLE
feat: add route for sellers to edit a profile

### PR DESCRIPTION
**Pull request by aapetsi**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works .
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by aapetsi** [comment]
[Seller profile Page](https://trello.com/c/TgiY85oW/33-seller-profile-page)

File: src/components/EditProfile/EditProfile.js
Comment (DimejiAre) [inline src/components/EditProfile/EditProfile.js]: same URL?

Comment (aapetsi) [reply src/components/EditProfile/EditProfile.js]: yeah one is  PUT request the other is a GET request

Comment (shaunorpen) [reply src/components/EditProfile/EditProfile.js]: Is it possible to just store URL and then use it for both? Would be good to avoid duplication if we can.

Comment (DimejiAre) [reply src/components/EditProfile/EditProfile.js]: If it's the same URL, do you need 2 variables?

Comment (aapetsi) [reply src/components/EditProfile/EditProfile.js]: i can change it to one url

File: package.json
Comment (shaunorpen) [inline package.json]: Why are we removing this? I think it's needed for other components, isn't it?

File: src/components/EditProfile/EditProfile.js
Comment (shaunorpen) [inline src/components/EditProfile/EditProfile.js]: We need to use AntD components here, don't we?

File: src/components/EditProfile/edit.css
Comment (shaunorpen) [inline src/components/EditProfile/edit.css]: Not sure we need this file. AntD should be used for most styling.

File: src/components/Login/Login.js
Comment (shaunorpen) [inline src/components/Login/Login.js]: Not sure we need this file. Have you pulled from develop recently?

File: src/components/EditProfile/edit.css
Comment (DimejiAre) [inline src/components/EditProfile/edit.css]: Move styling to less folder. That's where other styles are. You can create a less file for your component and import to index.less

**Review by shaunorpen — CHANGES_REQUESTED** [review]
I've raised a few questions below before I'm happy to approve. Happy to discuss if needed!

File: src/components/Login/Login.js
Comment (DimejiAre) [inline src/components/Login/Login.js]: Why did you create another Login.js?

File: package.json
Comment (aapetsi) [reply package.json]: this is strange. i didnt delete this file. i will reinstall it

File: src/components/Login/Login.js
Comment (aapetsi) [reply src/components/Login/Login.js]: I've pulled from develop already. Why dont we need the login file?

Comment (aapetsi) [reply src/components/Login/Login.js]: i didnt create a new one. there was a typo i fixed when i first cloned it. Seems the login file and folder is no longer being used

File: src/components/EditProfile/edit.css
Comment (aapetsi) [reply src/components/EditProfile/edit.css]: It was taking me a long time to come up to speed with ant-design so i styled everything with css. I'd be happy if u could walk me through changing it to ant design

File: src/components/Login/Login.js
Comment (DimejiAre) [reply src/components/Login/Login.js]: Okay. But right now on your branch, there's a login.js and a Login.js, and they have slightly different code. Maybe you should take a look at this and decide if one needs renaming or removing

Comment (aapetsi) [reply src/components/Login/Login.js]: alright i'll ask Shaun or Alex which file is being used

File: src/components/EditProfile/edit.css
Comment (Rexben001) [reply src/components/EditProfile/edit.css]: @shaunorpen

**Review by DimejiAre — CHANGES_REQUESTED** [review]
@aapetsi Looks good. One Major thing I think needs to be changed
1. The currency field should be a drop down. That's how it's being implemented during create store. This will help avoid the user entering any kind of value there.

Then some other minor fixes: 
1. The color of the buttons are blue. Other pages have black'ish' buttons.
2. There is no success prompt after update.
3. Logout button has not been implemented (should delete token and redirect to login page)

**Review by shaunorpen — CHANGES_REQUESTED** [review]
I'm uncomfortable that this appears to use standard HTML and CSS whereas other components use AntDesign components. Perhaps I'm looking in the wrong place? If not, is there a reason for the difference?

**Review by DimejiAre — APPROVED** [review]
@aapetsi Functionality is all good now. Don't know if Shaun has been able to help you with the ant design so that the component looks more uniform with the others.

File: src/App.js
Comment (shaunorpen) [inline src/App.js]: Perhaps update this so it's in the same format as the other Routes?

`<Route path='/profile'>
     <EditProfile />
</Route>`

File: src/components/EditProfile/EditProfile.js
Comment (shaunorpen) [inline src/components/EditProfile/EditProfile.js]: Can we delete these commented out lines?

File: src/components/EditProfile/EditProfile.js
Comment (shaunorpen) [inline src/components/EditProfile/EditProfile.js]: Can we increase the size of this text on the screen?

File: src/components/EditProfile/EditProfile.js
Comment (shaunorpen) [inline src/components/EditProfile/EditProfile.js]: Same for this text? It's really small...

File: src/less/edit.less
Comment (shaunorpen) [inline src/less/edit.less]: Can we delete these commented out lines?

Comment (shaunorpen) [reply src/less/edit.less]: Actually, is this file needed now that you've moved to AntD?

**Review by shaunorpen — CHANGES_REQUESTED** [review]
Great work! Just a few minor tweaks I'd like to suggest before I'm happy to approve!

File: src/less/edit.less
Comment (aapetsi) [reply src/less/edit.less]: yeah. i will use those to do some minor styling

**Review by shaunorpen — APPROVED** [review]

**Pull request merged**
